src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs

```diff
@@ -254,6 +254,7 @@ internal class BuiltinFunctionsCore
         public static readonly TexlFunction Decimal_UO = _featureGateFunctions.Add(new DecimalFunction_UO());
         public static readonly TexlFunction Float = _featureGateFunctions.Add(new FloatFunction());
         public static readonly TexlFunction Float_UO = _featureGateFunctions.Add(new FloatFunction_UO());
+        public static readonly TexlFunction IsEmpty_UO = _featureGateFunctions.Add(new IsEmptyFunction_UO());
         public static readonly TexlFunction IsUTCToday = _featureGateFunctions.Add(new IsUTCTodayFunction());

```

Comment:
This change is simple enough that we should just make it in Power Apps as well instead of diverging the platforms. Can you move it to the core _library instead?